colossalai/tensor/colo_tensor.py

```diff
@@ -296,3 +228,9 @@ def __mul__(self, other) -> "ColoTensor":
             raise TypeError(f'{type(other)} is not supported in ColoTensor __mul__')
 
     __rmul__ = __mul__
+
+    def to_dist_spec(self, dist_spec: _DistSpec) -> None:
+        self._torch_tensor = DistSpecManager.handle_trans_spec(self.torch_tensor(), self.spec.dist_spec, dist_spec)
+        # self._torch_tensor.requires_grad = self._requires_grad
+        self._size = self._torch_tensor.size()
```

Comment:
I think the attributes in current ColoTensor need to be defined clearly, like size,device.... Maybe they mean either local or global tensor info now.

Reply:
Shall we add two methods like `is_global()` and `to_global()` ? In this way, we assume ColoTensor is local by default. If you have any good ideas, please let me know.

Reply:
Make sense. I added a global_torch_tensor() in ColoTensor before to get tensors in global view, but it only serves for activations. Maybe we can re-implement it with dictspec.

Reply:
We can do this in the next PR.